src/inmanta/deploy/state.py

```diff
@@ -144,8 +144,8 @@ class BlockedStatus(StrEnum):
         Concretely it is waiting for its dependencies to deploy successfully.
     """
 
-    YES = enum.auto()
-    NO = enum.auto()
+    BLOCKED = enum.auto()
+    FREE = enum.auto()
```

Comment:
rename 7.

motivation: YES/NO always felt clunky to me.

notes:
- not sure about `FREE`. Could also be `NOT_BLOCKED`
- will require db migration script

personal opinion: on the fence, input welcome.

Reply:
I also didn't like the YES/NO.

Reply:
* If we are going to change this, I wonder whether we should use `PERMANENTLY_BLOCKED` instead of `BLOCKED`. It would make more clear what it represents.
* I am not convinced that `FREE` is any better than `NO`, but I have no objections against the rename.

Reply:
But do you prefer the new proposal?

Reply:
> If we are going to change this, I wonder whether we should use PERMANENTLY_BLOCKED instead of BLOCKED. It would make more clear what it represents.

I like the suggestion, though I think "permanent" is too... well, permanent. I still prefer the simple `BLOCKED` overall, but perhaps we can find a middle ground.

> I am not convinced that FREE is any better than NO

Me neither tbh. Wdyt of `NOT_BLOCKED` versus `FREE` and `NO`?

Reply:
I think I like `FREE` the least of all. I don't have a strong preference amongst. `NON_BLOCKED` or `NO`. Maybe a slight preference for `NON_BLOCKED`, because it's consistent with `BLOCKED`.

Reply:
> I like the suggestion, though I think "permanent" is too... well, permanent.

I see your point. I can't immediately think of a better name so maybe we have to stick with `BLOCKED` after all.

Reply:
What would you think of `TRANSIENT -> TEMPORARILY_BLOCKED`? It's more verbose than I'd like, but it's consistent, and perhaps more clear than transient (especially considering that transitive sounds so similar and is also an important part of the blocking).

Reply:
> What would you think of TRANSIENT -> TEMPORARILY_BLOCKED?

I like it a lot. I also agree on the verbosity remark, but I am not afraid of using longer names if it makes things more clear.

Reply:
Ok, new consolidated proposal for this one:
```python
class Blocked(Enum):
    BLOCKED
    NOT_BLOCKED
    TEMPORARILY_BLOCKED
```

Wdyt? (@wouterdb as well)